home.css

```diff
@@ -0,0 +1,811 @@
+body {
+  color: #595c61;
+}
+
+nav {
+  display: none;
+}
+
+#close-button {
+  display: none;
+  position: relative;
+  top: -90%;
+  left: 85%;
+  color: #ec5242;
+}
+
+header {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  background-image: url(./images/background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 20px 20px 50px 20px;
+}
+
+#header-section {
+  margin-top: 35px;
+}
+
+#menu {
+  color: #595c61;
+}
+
+#salutation {
+  color: #ec5242;
+  font-family: lato, sans-serif;
+  margin: 30px 0 5px 0;
+  font-size: 1.2rem;
+}
+
+#header-section h1 {
```

Comment:
- [ ] For your `header-section` div please fix all the elements in it adding the correct font size and font-weight for each element to match the given design.
- [ ] Also please add the value of the correct margins between all those elements too and try to stick as much as possible to the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/169351814-8f162abd-43f7-4efd-8d98-d204ca4b71ba.png)  | ![image](https://user-images.githubusercontent.com/68796232/169351764-46aa7601-77e2-4d3e-a0de-89ffc2f391a2.png) |

Reply:
Thank you @Omar-Muhamad. I adjusted accordingly.